### DISCUSSION TITLE
Option to change set a custom song after the intro?

### DISCUSSION THREAD

**Discussion by mandarinka12051** [ppy/osu#26729] [Ideas]
You can change the intro sequence between triangles, circles and welcome. You can also make the default music not play and choose a random song instead. I know that, but i wish that there would be a feature that after the circles or welcome intro, you could select a song from your beatmap list that plays everytime after the intro after you launch osu!lazer. To clarify, simply set your own song instead of "circles!" and "Welcome to osu" songs. I excluded the triangles one because the music is also involved in the intro, so that would give extra work. But Welcome to osu and circles! songs aren't, so that's why i want them when you have circles or welcome intro sequence.

**Comment by Rekunan** [comment]
I have asked for this before, and peppy said that it's out of scope for the game: https://github.com/ppy/osu/discussions/15169#discussioncomment-1490993
